extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -7,17 +7,21 @@ import * as azdata from 'azdata';
 import * as vscode from 'vscode';
 import * as path from 'path';
 import * as fs from 'fs-extra';
+import * as nls from 'vscode-nls';
 import { IPrompter, QuestionTypes, IQuestion } from '../prompts/question';
 import CodeAdapter from '../prompts/adapter';
 import { BookTreeItem } from './bookTreeItem';
 import { BookModel } from './bookModel';
 import { Deferred } from '../common/promise';
+import { IBookTrustManager, BookTrustManager } from './bookTrustManager';
 import * as loc from '../common/localizedConstants';
 
+const localize = nls.loadMessageBundle();
+const msgBookTrusted = localize('msgBookTrusted', "Book is now automatically trusted.");
```

Comment:
It's preferable to put all strings in the https://github.com/microsoft/azuredatastudio/blob/master/extensions/notebook/src/common/localizedConstants.ts file